Resources/public/js/views/fields/ez-selection-editview.js

```diff
@@ -64,6 +64,7 @@ YUI.add('ez-selection-editview', function (Y) {
                 this._set('values', this._getSelectedTextValues());
             });
             this.after('valuesChange', this._uiSyncSelectionValues);
+            this.after('valuesChange', this.checkValueChange);
```

Comment:
the name of the method is bad because if I just read this line, I have absolutely no clue of what it does.